sky/provision/__init__.py

```diff
@@ -4,13 +4,15 @@
 providers supported by SkyPilot need to follow.
 """
 import functools
-import importlib
 import inspect
 from typing import Any, Dict, List, Optional
 
 from sky import sky_logging
 from sky import status_lib
+from sky.provision import aws
+from sky.provision import azure
 from sky.provision import common
+from sky.provision import gcp
```

Comment:
Should add a comment on: ...these imports should never fail even if underlying cloud SDK dependencies are not installed. This is ensured by using the .... pattern inside these modules.